source/loop/apis.rst

```diff
@@ -914,6 +914,57 @@ POST /rooms
             "roomUrl": "http://localhost:3000/static/#rooms/pPVoaqiH89M"
         }
 
+PATCH /rooms
+~~~~~~~~~~~~
+
+    **Requires owner authentication**
+
+    Remove given rooms
+
+    Request body parameters:
+
+    - **deleteRoomTokens**, a list of rooms to delete.
+
+    Response body parameters:
+
+    - **deletedRoomTokens**, a list of room's tokens actually deleted.
```

Comment:
This doesn't match what is implemented. The current implementation return `responses`.  see https://github.com/mozilla-services/loop-server/pull/272/files#diff-2fbbe66e501e7e82cd4c1868a4632212R525